### ISSUE TITLE
new term: papillitis

### ISSUE THREAD

**Issue by mellybelly** [obophenotype/human-phenotype-ontology#7509]
**Preferred term label:**
papillitis

**Synonyms**
inflammation of the optic disc

**Definition (free text, please give PubMed ID)**
Inflammation of the optic disc in the absence of ischemia and malignant infiltration.

**Parent term (use hpo.jax.org/app)**
Abnormality of the optic disc HP:0012795

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Birdshot disease 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Parent 
[Optic neuritis HP:0100653](https://hpo.jax.org/app/browse/term/HP:0100653)
def: Optic papillitis is a form of optic neuritis localized at the optic nerve head. Papillitis refers to optic neuritis with involvement of the intraocular portion of the optic nerve with exudates and hemorrhages on and around the disc. This is in contrast to retrobulbar neuritis, which is optic neuritis of a more posterior portion of the nerve, and which, in the early stage, produces no ophthalmoscopic visible evidence. Papillitis may have the same appearance as papilledema. However, papilledema is more often bilateral and optic papillitis unilateral. A sudden loss of visual acuity speaks for papillitis, while gradual or little to no loss speaks for papilledema. Papilledema is classically characterized by an enlarged blindspot. Optic papillitis can be characterized by a central or paracentral scotoma together with some enlargement of the blindspot. Papillitis is more commonly associated with an afferent pupillary defect (Marcus Gunn pupil).
PMID:22888383

Synonym: Intraocular optic neuritis, Optic neuritis with optic disc swelling

@sukramg

**Comment by Sukramg** [comment]
Papillitis is a very fuzzy term and I try to avoid it. You notice that quite quickly when you type the term into Pubmed. Typically, it is a unilateral papillary swelling due to infection.  Clinically, papillitis cannot be distinguished with certainty from papilledema. 

It could be described this way: 
Papillitis is inflammation of the optic nerve at its exit point from the eye bulb (optic nerve papilla) with papilledema. The inflammation can lead to acute loss of vision.

Aetiology
Triggering factors of papillitis can be infectious diseases, progressive inflammations, intoxications, allergic-hyperergic and immunological processes.

In practice, therefore, it is the other way round: If I have evidence of an inflammatory genesis "itis", I call the papilloedema papillitis, typically in the case of uveitis.

**Preferred term label:**
papillitis

**Synonyms**
inflammation of the optic disc, intraocular optic neuritis, Optic neuritis with optic disc swelling



**Definition (free text, please give PubMed ID)**
def: Optic papillitis is a form of optic neuritis localized at the optic nerve head. Papillitis refers to optic neuritis with involvement of the intraocular portion of the optic nerve with exudates and hemorrhages on and around the disc. This is in contrast to retrobulbar neuritis, which is optic neuritis of a more posterior portion of the nerve, and which, in the early stage, produces no ophthalmoscopic visible evidence. Papillitis may have the same appearance as papilledema. However, papilledema is more often bilateral and optic papillitis unilateral. A sudden loss of visual acuity speaks for papillitis, while gradual or little to no loss speaks for papilledema. Papilledema is classically characterized by an enlarged blindspot. Optic papillitis can be characterized by a central or paracentral scotoma together with some enlargement of the blindspot. Papillitis is more commonly associated with an afferent pupillary defect (Marcus Gunn pupil).
PMID:22888383

**Parent term (use hpo.jax.org/app)**
[Optic neuritis HP:0100653](https://hpo.jax.org/app/browse/term/HP:0100653)
def: Optic papillitis is a form of optic neuritis localized at the optic nerve head. Papillitis refers to optic neuritis with involvement of the intraocular portion of the optic nerve with exudates and hemorrhages on and around the disc. This is in contrast to retrobulbar neuritis, which is optic neuritis of a more posterior portion of the nerve, and which, in the early stage, produces no ophthalmoscopic visible evidence. Papillitis may have the same appearance as papilledema. However, papilledema is more often bilateral and optic papillitis unilateral. A sudden loss of visual acuity speaks for papillitis, while gradual or little to no loss speaks for papilledema. Papilledema is classically characterized by an enlarged blindspot. Optic papillitis can be characterized by a central or paracentral scotoma together with some enlargement of the blindspot. Papillitis is more commonly associated with an afferent pupillary defect (Marcus Gunn pupil).
PMID:22888383

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
neuroretinitis; http://purl.obolibrary.org/obo/MONDO_0000958

Your nano-attribution (ORCID)
0000-0002-6601-4486

**Comment by pnrobinson** [comment]
After discussions with @Sukramg it seems that the actual phenotypic appearance of papillitis cannot be distinguished from papilledema, and basically the "itis" is an inference from other signs in the examination. This means that this is probably not a good HPO term but rather could be a related synonym of papilledema.

**Comment by pnrobinson** [comment]
added as RELATED SYNONYM
